test/routes/src/server/routes/t_webhook.js

```diff
@@ -307,20 +307,16 @@ describe('The WebHook API endpoint', () => {
               });
             requestAutoBuilds = nock(lp_api_url)
               .post(`/devel${lp_snap_path}`, { 'ws.op': 'requestAutoBuilds' })
-              .reply(200, {
-                total_size: 2,
-                start: 0,
-                entries: [
-                  {
-                    resource_type_link: `${lp_api_base}/#snap_build`,
-                    self_link: `${lp_api_base}${lp_snap_path}/+build/1`
-                  },
-                  {
-                    resource_type_link: `${lp_api_base}/#snap_build`,
-                    self_link: `${lp_api_base}${lp_snap_path}/+build/2`
-                  }
-                ]
-              });
+              .reply(200, [
+                {
```

Comment:
Why is this change added?
Doesn't LP collection response contain `total_size` etc properties?

Reply:
Not the response of `requestAutoBuilds`, it simply returns the created builds in the body. It must have been confused with the response of snap builds which is indeed a LP collection with total_size and start.